src/chart/treemap/TreemapView.ts

```diff
@@ -979,6 +979,9 @@ function renderNode(
             }
         );
 
+        if (!isShow) {
```

Comment:
textEl is also needed when label is set `emphasis.show = true`. Perhaps we can check if `textEl` is null in the next line instead of check if show on normal state.

Reply:
Right, I missed that.

Reply:
@pissang The reason is in `labelStyle`, a text element needs to be create should consider all four state, but in `treemap`, it only consider `normal` state.

Reply:
@susiwen8 Year we need to check all four states. But it has been done in `setLabelStyle`. text element will be created if any state is set `show: true`. So I think we only need to check if `rectEl.getTextContent()` exists.

Reply:
@pissang Thx, You are right.